spec/type.dd

```diff
@@ -30,9 +38,9 @@ $(H2 $(LEGACY_LNAME2 Basic Data Types, basic-data-types, Basic Data Types))
     $(TROW $(D cfloat), $(D float.nan+float.nan*1.0i), a complex number of two float values)
     $(TROW $(D cdouble), $(D double.nan+double.nan*1.0i), complex double)
     $(TROW $(D creal), $(D real.nan+real.nan*1.0i), complex real)
-    $(TROW $(D char), $(D 0xFF), unsigned 8 bit (UTF-8 code unit))
-    $(TROW $(D wchar), $(D 0xFFFF), unsigned 16 bit (UTF-16 code unit))
-    $(TROW $(D dchar), $(D 0x0000FFFF), unsigned 32 bit (UTF-32 code unit))
+    $(TROW $(D char), $(D '\xFF'), unsigned 8 bit (UTF-8 code unit))
+    $(TROW $(D wchar), $(D '\uFFFF'), unsigned 16 bit (UTF-16 code unit))
+    $(TROW $(D dchar), $(D '\U0000FFFF'), unsigned 32 bit (UTF-32 code unit))
```

Comment:
LaTeX offers `\textbackslash`, but AFAICT it hasn't been used before, so you would need to do sth. like:

spec/latex.ddoc
```
BACKSLASH=\textbackslash
```

spec/dlang.org.ddoc and ebook.ddoc:
```
BACKSLASH=\
```

(I don't like the Ddoc setup either)

Reply:
Curiously, `\x...` is used in lex.dd